test/verify/check-shell-menu

```diff
@@ -204,6 +204,16 @@ class TestMenu(testlib.MachineCase):
         # different browsers behave a bit different here -- Firefox is at "Overview", Chromium at "Logs"
         b.wait_js_cond("document.activeElement.classList.contains('pf-v5-c-nav__link')")
 
+    def testCustomLandingPage(self):
+        b = self.browser
+        m = self.machine
+
+        # # Make the logs page the default page
+        m.write("/etc/cockpit/systemd.override.json", '{"menu":{"index":{"order":100},"logs":{"order":1}}}')
```

Comment:
This is nondestructive, so self.write_file() please.

Reply:
We restore_dir("/etc/cockpit") in test/common/testlib.py

Reply:
Oh sorry, indeed!